op-e2e/actions/user.go

```diff
@@ -317,6 +319,90 @@ func (s *CrossLayerUser) CheckDepositTx(t Testing, l1TxHash common.Hash, index i
 	}
 }
 
+func (s *CrossLayerUser) ActStartWithdrawal(t Testing) {
+	targetAddr := common.Address{}
+	if s.L1.txToAddr != nil {
+		targetAddr = *s.L2.txToAddr
+	}
+	tx, err := s.L2.env.Bindings.L2ToL1MessagePasser.InitiateWithdrawal(&s.L2.txOpts, targetAddr, new(big.Int).SetUint64(s.L1.txOpts.GasLimit), s.L1.txCallData)
+	require.NoError(t, err, "create initiate withdraw tx")
+	err = s.L2.env.EthCl.SendTransaction(t.Ctx(), tx)
```

Comment:
Should this wait on receipt?

Reply:
Also a separate action, so we can send multiple txs and do other actions, before the block is sealed and receipts are available